tasks/setup/InfracostSetup/index.ts

```diff
@@ -0,0 +1,284 @@
+import * as taskLib from 'azure-pipelines-task-lib/task';
+import * as toolLib from 'azure-pipelines-tool-lib/tool';
+import * as path from 'path';
+import * as os from 'os';
+import * as semver from 'semver';
+import fetch from 'node-fetch';
+
+/**
+ * archMappings defines how different https://nodejs.org/api/os.html#os_os_arch
+ * should be represented in terms of the infracost cli assets.
+ */
+const archMappings: { [s: string]: string } = {
```

Comment:
👍

Reply:
@vdmgolub like the idea of using the airbnb styleguide. Thanks for bringing to my attention.

Reply:
@vdmgolub though in this case I think we should leave this const as is. The guide states:

>  only if it (1) is exported, (2) is a const (it can not be reassigned), and (3) the programmer can trust it (and its nested properties) to never change.

This const isn't exported

Reply:
Ah, my bad. Let's keep them as is then 👍